Documentation/LanguageServer/c_cpp_properties.json.md

```diff
@@ -45,43 +45,45 @@
+  An include path is a folder that contains header files (such as `#include "myHeaderFile.h"`) that are included in a source file. Specify a list paths for the IntelliSense engine to use while searching for included header files. If a path ends with `/**` the IntelliSense engine will do a recursive search for hearder files starting from that directory. If on Windows with Visual Studio installed, or if a compiler is specified in the `compilerPath` setting, it is not necessary to list the system include paths in this list.
 
 * #### `defines`
-  If `"C_Cpp.intelliSenseEngine"` is set to "Default" in your settings file, this list of preprocessor symbols will be used by IntelliSense during the compilation of your source files. This is basically the same as the list of symbols you pass to your compiler with the `-D` switch. If on Windows with Visual Studio installed, or if a compiler is specified in the `compilerPath` setting, it is not necessary to list the system include paths in this list.
-
-* #### `forcedInclude` (optional)
-  A list of files that should be included before any other characters in the source file are processed. Files are included in the order listed.
-
-* #### `compilerPath` (optional)
-  The absolute path to the compiler you use to build your project. The extension will query the compiler to determine the system include paths and default defines to use for IntelliSense.
-  
-  Args can be added to modify the includes/defines used, e.g. `-nostdinc++`, `-m32`, `-fno-ms-extensions`, etc., but paths with spaces must be surrounded by double quotes (`"`) if args are used.
-  
-  If you want to disable automatic querying of system include paths and defines, set this value to the emptry string `""`. This is generally not recommended, but there are some cases where automatic querying of system includes and defines is undesirable.
+  A list of preprocessor definitions for the IntelliSense engine to use while parsing files. Optionally, use `=` to set a value, e.g. `VERSION=1`.
 
 * #### `cStandard`
-  The C standard revision to use for IntelliSense in your project.
+  The version of the C language standard to use for IntelliSense.
 
 * #### `cppStandard`
-  The C++ standard revision to use for IntelliSense in your project.
+  The version of the C++ language standard to use for IntelliSense.
+
+* #### `configurationProvider`
+  The ID of a VS Code extension that can provide IntelliSense configuration information for source files.
+
+* #### `windowsSdkVersion`
+  The version of the Windows SDK include path to use on Windows, e.g. `10.0.17134.0`.
+
+* #### `macFrameworkPath`
+  A list of paths for the Intellisense engine to use while searching for included headers from Mac frameworks. Only supported on Mac configuration.
```

Comment:
"Mac configuration" seems like it should be "macOS".

Reply:
How about "Mac platform"? `Only supported on configurations for Mac platform.`

Reply:
I think the correct term is "macOS". I'm not sure what "Mac platform" means (if you really want to use that term, it should be "the Mac platform"). https://en.wikipedia.org/wiki/MacOS . For legacy reasons we still use "Mac" in various places.

Reply:
Okay, I changed any 'Mac' to 'macOS'.